admin/class-automation.php

```diff
@@ -230,42 +258,47 @@ public static function get_automation_rules(): array {
 	 */
 	public static function get_fields(): array {
 		return [
-			'isHidden'       => [
+			'contentGenerationType' => [
+				'location' => 'metadata',
+				'type'     => 'string',
+				'label'    => __( 'contentGenerationType', 'apple-news' ),
```

Comment:
if the text is a specific meta key and should always be that value, do we want to translate it?